Find404.scala

```diff
@@ -1,16 +1,20 @@
 import java.io._
 import scala.io.Source._
 import scala.io.StdIn._
+import scala.collection.mutable.ArrayBuffer
 
-object Find404 {
-    
-    def main(args: Array[String]): Unit = {
-        println("Provide a directory")
-        val directory = new File(readLine())
+val fileSet = ArrayBuffer[File]()
 
-        val directorySet = directory.listFiles()
-        val fileSet = directorySet.filter(_.getClass().getName().indexOfSlice(".") == -1)
+try {
+    val directorySet = new File(args(0))
+    exploreDirect(directorySet, fileSet)
+} catch {
+    case _: NullPointerException => println("Not a valid directory")
```

Comment:
There are ways to make sure an exception doesn't happen. Doing so is better than catching an exception. [Browse the file api](https://docs.oracle.com/javase/8/docs/api/java/nio/file/Files.html#exists-java.nio.file.Path-java.nio.file.LinkOption...-)

Reply:
There are things like `exists`, `isDirectory`, etc